microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,601 @@
+
+        if (Url != string.Empty)
+        {
+            Debug.Log("Loading URL: " + Url);
+            webViewComponent.Load(new Uri(Url));
+        }
+
+        webViewComponent.GetWebViewWhenReady((IWebView webView) =>
+        {
+            // Add event listeners for WebView2 events
+            ((IWithPostMessage)webView).MessageReceived += OnMessageReceived;
+        });
+    }
+
+    // Handler for WebView2 OnPostMessage event
+    void OnMessageReceived(string message)
+    {
+        Debug.Log("WebView2 message received: " + message);
+    }
+}
+```
+
+<!-- ====================================================================== -->
+## Additional considerations for WebView2 on HoloLens 2
+
+
+<!-- ------------------------------ -->
+#### Limitations and known issues
+
+When developing a HoloLens 2 Unity app with WebView2, be aware of some limitations and known issues:
+
+* **Pop-ups**: Pop-ups don't work well within WebView2 inside Unity apps on HoloLens 2, but they work fine in 2D XAML apps on the device. Avoid pop-ups and use alternative techniques or UI designs, such as custom pop-up-like elements within the WebView using HTML, CSS, and JavaScript.
+
+* **New windows**: WebView2 instances on HoloLens 2 navigate within the same window by default, unlike on Desktop. Follow this default behavior for a better user experience.
+
+* **Enterprise authentication**: Automatic Single Sign-On (SSO) leveraging OS-level tokens is currently not supported in WebView2 on HoloLens 2. Users can still sign in by providing credentials, except for cases requiring device-level authentication. Cookie storage works as expected.
+
+* **User interactions**: Unlike native HoloLens 2 slates, WebView2 is best interacted with using far-interaction hand rays. Touch-to-swipe and scroll interactions might not be supported.
+
+* **Performance**: Complex websites with heavy use of JavaScript or advanced rendering may impact system performance or the host application's framerate. For general performance-related limitations and recommendations, see [Understanding performance for mixed reality](/windows/mixed-reality/develop/advanced-concepts/understanding-performance-for-mixed-reality) in the Mixed Reality documentation. Also see [Performance optimization](#performance-optimization), below.
+
+
+<!-- ------------------------------ -->
+#### Navigation
+
+In Step 6, we touched on some navigation methods. In this section, we'll expand on what we learned.
+
+
+<!-- ---------- -->
+###### IWebView interface
+
+<!-- [IWebView::Load method]()-->
+
+The `IWebView` interface exposes a few methods, events, and properties related to page navigation. The main functionality exposed here is the ability to navigate to a given URL, by using `Load(Uri url)`:
+
+```C#
+public interface IWebView
+{
+    // Non-navigation methods are removed for clarity.
+
+    event WebView_OnNavigated Navigated;
+
+    Uri Page { get; }
```

Comment:
```suggestion
    Uri Page { get; };
```
@michaelfarnsworth - tech review this change

Reply:
Not needed.